test/cctest/test_util.cc

```diff
@@ -253,7 +258,7 @@ TEST(UtilTest, MaybeStackBuffer) {
   }
 }
 
-TEST(UtilTest, SPrintF) {
+TEST_F(UtilTest, SPrintF) {
```

Comment:
Do you need to change the others to `TEST_F`? Wouldn't it be just adding a new test with `TEST_F`?

Reply:
It doesn't let you mix both in the same file. I could create a new file but that didn't seem coherent